perception/radar_object_tracker/src/radar_object_tracker_node/radar_object_tracker_node.cpp

```diff
@@ -252,6 +415,51 @@ void RadarObjectTrackerNode::checkTrackerLifeCycle(
   }
 }
 
+// remove objects by lanelet information
+void RadarObjectTrackerNode::mapBasedNoiseFilter(
+  std::list<std::shared_ptr<Tracker>> & list_tracker, const rclcpp::Time & time)
+{
+  for (auto itr = list_tracker.begin(); itr != list_tracker.end(); ++itr) {
+    autoware_auto_perception_msgs::msg::TrackedObject object;
+    (*itr)->getTrackedObject(time, object);
+    const auto closest_lanelets = getClosestValidLanelets(
+      object, lanelet_map_ptr_, max_distance_from_lane_, max_angle_diff_from_lane_);
+
+    // 1. If the object is not close to any lanelet, delete the tracker
+    const bool no_closest_lanelet = closest_lanelets.empty();
+    // 2. If the object velocity direction is not close to the lanelet direction, delete the tracker
+    const bool is_velocity_direction_close_to_lanelet =
+      hasValidVelocityDirectionToLanelet(object, closest_lanelets, max_lateral_velocity_);
+    if (no_closest_lanelet || !is_velocity_direction_close_to_lanelet) {
+      // std::cout << "object removed due to map based noise filter" << " no close lanelet: " <<
+      // no_closest_lanelet << " velocity direction flag:" << is_velocity_direction_close_to_lanelet
+      // << std::endl;
+      itr = list_tracker.erase(itr);
+      --itr;
+    }
+  }
+}
+
+// remove objects by distance
+void RadarObjectTrackerNode::distanceBasedNoiseFilter(
```

Comment:
@coderabbitai Would it be easier to understand the method here if we start with the verb?